src/picologging/handler.cxx

```diff
@@ -70,7 +70,13 @@ PyObject* Handler_handle(Handler *self, PyObject *record) {
     }
     
     self->lock->unlock();
-    return result;
+
+    // emit failed
```

Comment:
You could write this as a ternary expression instead, I think something like:

```cpp
return result == nullptr ? nullptr : Py_True;
```

Reply:
Thanks, done.